tools/ds-identify

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
@TheRealFalcon @blackboxsw 
This script is using `local` keyword in almost all functions & `local` is not POSIX compliant (shellcheck warns the same).
Can we use bash by default? Ubuntu links sh->dash and I'm unsure about other systems.
I'm assuming that bash is available in all distros by default.

Reply:
I would focus this PR on improving `dscheck_VMware` and separate general script improvements into a separate change.

Reply:
I think I got my answer by CI test results. It will be nice to know your clarification.